ux.symfony.com/cookbook/component_architecture.md

```diff
@@ -11,25 +11,25 @@ published_at: '2024-08-02'
 
 ## Introduction
 
-In Symfony UX exist two packages: [TwigComponents](https://symfony.com/bundles/ux-twig-component/current/index.html) and [LiveComponent](https://symfony.com/bundles/ux-live-component/current/index.html).
-Those two packages allow you to create reusable components in your Symfony application.
-But the component architecture is not exclusive to Symfony, it's a design pattern that can be applied to any programming language or framework.
-And the Javascript world already implements this architecture for long time, on many different frameworks like React, Vue, or Svelte.
-So, a set of rules and patterns has already be defined to work with components. This is why Symfony UX tries to be as close as possible to those rules.
-So, let's see what those rules are!
+In Symfony UX, there are two packages: [TwigComponents](https://symfony.com/bundles/ux-twig-component/current/index.html) and [LiveComponent](https://symfony.com/bundles/ux-live-component/current/index.html).
```

Comment:
General question, should we backtick TwigComponents and LiveComponent?

Reply:
Also should use plural on both or none.